src/stores/my-account.ts

```diff
@@ -27,23 +49,35 @@ type Session = {
   isNewSecretKey?: boolean
 }
 type Actions = {
-  login: (
+  isAuthenticated?: boolean
+  loginAnonymously: (
     secretKey?: string,
     isInitialization?: boolean
   ) => Promise<string | false>
-  logout: () => void
+  initializeWeb3Auth: () => Promise<Web3Auth>
+  login: () => Promise<void>
+  logout: () => Promise<void>
+  getPrivateKey: () => Promise<string | undefined>
```

Comment:
with you getting private key (related to previous comments), this is not needed

Reply:
alright, will move the function into the web3authlogin